charts/k8ssandra/templates/reaper/reaper-jmx-secret.yaml

```diff
@@ -14,6 +22,6 @@ data:
   {{- if .Values.reaper.jmx.password }}
   password: {{ .Values.reaper.jmx.password | b64enc | quote }}
   {{- else }}
-  password: {{ include "k8ssandra-common.password" . }}
+  password: {{ $password }}
```

Comment:
So this tackles the password, but is the username guaranteed to not be regenerated?

Reply:
Nice catch :) I missed that. I will update that now.